context/app/static/js/components/files/FileBrowserFile/style.js

```diff
@@ -12,7 +12,7 @@ const StyledDiv = styled.div`
 `;
 
 const IndentedDiv = styled.div`
-  padding: 10px ${(props) => props.theme.spacing(3)}px;
+  padding: 10px 40px;
```

Comment:
Should there be some way of linking the 40 here and above, even if it's not `props.theme.spacing`?

Reply:
Filed #1142 as I will have to shift a few things around.